examples/README.md

```diff
@@ -1542,10 +1542,15 @@ cleaned up when the job is deleted.
 This blueprint shows how to provision a GKE cluster with A3 Mega machines in the toolkit.
 
 After provisioning the cluster and the nodepool, we need to do the following:
+
 1- Install the GPUDirect binary and configure NCCL: https://cloud.google.com/kubernetes-engine/docs/how-to/gpu-bandwidth-gpudirect-tcpx#install-gpudirect-tcpx-nccl
```

Comment:
If you use `1.` instead of `1-` markdown will auto format into list, and you won't need white space between list items.

Reply:
done